test/metabase/lib/schema/expression/arithmetic_test.cljc

```diff
@@ -0,0 +1,33 @@
+(ns metabase.lib.schema.expression.arithmetic-test
+  (:require
+   [clojure.test :refer [deftest is testing]]
+   [malli.core :as mc]
+   [metabase.lib.schema.expression :as expression]
+   [metabase.lib.schema.expression.arithmetic]
```

Comment:
Side-effect requires on different namespaces full of schemas is pretty tedious.
Can't we move any shared internals to `metabase.lib.schema.expression.base` and make `metabase.lib.schema.expression` just import all the other `.schema.expression.*` namespaces?

Reply:
Yes, I agree. One of the reasons for using qualified schema in the first place was to force people to `require` the namespaces where they were loaded, unfortunately with the new way things work that's less of a thing now.

I think I'll address this in a follow-on PR and rework the ways the schemas work a bit, I think what you're suggesting is a good idea. I might just try to do that for the entire schema rather than just for `expression.*`. And at that point we might want to consider dropping the fully-qualified names, or just prefix everything with the base schema's name, regardless of where they're defined